File: algoritmos/03 - Estrutura condicional/ec_02/Questões/questao02.py
# Questão 02

numero1 = int(input("Digite o 1º número: "))
numero2 = int(input("Digite o 2º número: "))
numero3 = int(input("Digite o 3º número: "))

maior = 0
menor = 0

# Condicional maior
if numero1 > numero2 and numero1 > numero3:
    maior = numero1
elif numero2 > numero1 and numero2 > numero3:
    maior = numero2
else:
    maior = numero3

# Condicional menor
if numero1 < numero2 and numero1 < numero3:
    menor = numero1
elif numero2 < numero1 and numero2 < numero3:
    menor = numero2
else:
    menor = numero3

print(f"Maior: {maior}")
print(f"Menor: {menor}")

File: estrutura de dados/3 - Funções/exercicios/questao02.c
/*
  ********************************************
  *        Victória Mendes da Costa          *
  *        Lic. em Computação - IFPI         *
  *        Estrutura de Dados - 2024.2       *
  ********************************************
*/

#include <stdio.h>

int maior(int a, int b) {
    return (a > b) ? a : b;
}

int main() {
    int numero1, numero2;

    printf("Digite o primeiro número: ");
    scanf("%d", &numero1);

    printf("Digite o segundo número: ");
    scanf("%d", &numero2);

    printf("O maior número entre %d e %d é %d\n", numero1, numero2, maior(numero1, numero2));

    return 0;
}